cmake/scripts/common/Macros.cmake

```diff
@@ -667,7 +667,7 @@ macro(core_find_versions)
   include(CMakeParseArguments)
   core_file_read_filtered(version_list ${CORE_SOURCE_DIR}/version.txt)
   core_file_read_filtered(json_version ${CORE_SOURCE_DIR}/xbmc/interfaces/json-rpc/schema/version.txt)
-  string(REPLACE " " ";" version_list "${version_list} ${json_version}")
+  string(REGEX REPLACE "([^ ;]*) ([^;]*)" "\\1;\\2" version_list "${version_list};${json_version}")
```

Comment:
are you saying there are blanks in some versions?

Reply:
there are blanks in `COMPANY_NAME` (see fixed version.txt)